### PR TITLE
Add suffixes to typos that start with "u"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -53286,6 +53286,8 @@ tyrany->tyranny
 tyring->trying
 tyrranies->tyrannies
 tyrrany->tyranny
+uability->usability
+uable->unable, usable,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53290]: Perhaps `able`, `suable` and `table` (and even `cable`, `fable`, `gable`, `sable`).

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -53324,6 +53337,13 @@ udpating->updating
 ue->use, due,
 ueful->useful
 uegister->unregister
+ueried->queried
+ueries->queries
+uername->username
+uernames->usernames
+uers->users, user,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:53344]: Also `hers`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -54574,6 +54646,9 @@ upsupported->unsupported
 uptadeable->updatable
 uptdate->update
 uptim->uptime
+uption->option
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:54649]: I guess `eruption` is too distant, isn't it?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -54649,7 +54729,26 @@ usnig->using
 uspported->supported, unsupported,
 usptart->upstart
 usptarts->upstarts
+usre->user, sure, use,
+usred->used
+usrename->username
+usrenames->usernames
+usrer->user, surer,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:54736]: Also `usurer` and `usher` (plural/other verbal forms too)?

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:53344]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:53290]: I'll add `table` and `able`. I couldn't find the others.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**